x-pack/plugins/security_solution/public/management/components/endpoint_responder/lib/console_commands_definition.ts

```diff
@@ -501,6 +520,48 @@ export const getEndpointConsoleCommands = ({
     });
   }
 
+  if (isScanEnabled) {
+    consoleCommands.push({
+      name: 'scan',
+      about: getCommandAboutInfo({
+        aboutInfo: CONSOLE_COMMANDS.scan.about,
+        isSupported: doesEndpointSupportCommand('scan'),
+      }),
+      RenderComponent: ScanActionResult,
+      meta: {
+        agentType,
+        endpointId: endpointAgentId,
+        capabilities: endpointCapabilities,
+        privileges: endpointPrivileges,
+      },
+      exampleUsage: 'scan --path "/full/path/to/folder" --comment "Scan folder for malware"',
+      exampleInstruction: ENTER_OR_ADD_COMMENT_ARG_INSTRUCTION,
+      validate: capabilitiesAndPrivilegesValidator(agentType),
+      mustHaveArgs: true,
+      args: {
+        path: {
+          required: true,
+          allowMultiples: false,
```

Comment:
suggestion: you should add `mustHaveValue: 'non-empty-string'`

refernce: https://github.com/elastic/kibana/blob/f9f4c1a8009e03f6e0f46879dc912b7baa9e3a9f/x-pack/plugins/security_solution/public/management/components/console/types.ts#L38-L51

Reply:
99e13666cb030e03427e48b3bf935b09b293207d